app/experimenter/experiments/tasks.py

```diff
@@ -184,6 +188,18 @@ def update_status(experiment):
             bugzilla.update_bug_resolution(experiment)
 
 
+def send_experiment_ending_emails(experiment):
+    # send experiment ending soon emails if end date is 5 days out
+    if experiment.end_date - date.today() <= timedelta(days=5):
```

Comment:
It's a small little thing but I like to use parens to denote order of operations on lines like this.  Python is pretty smart about figuring out what order you meant, but it's still better to be explicit.